Dockerfile

```diff
@@ -40,50 +40,61 @@ RUN HOME=/home/default && \
     usermod -a -G sudo default && \
```

Comment:
check the `Dockerfile`

Reply:
I get two errors:

1. `./scripts/docker-build.sh: line 3: ${$1:-develop}: bad substitution`
2. Missing `CUDD_INCLUDE_DIRS` in Lydia configuration, `Could NOT find CUDD (missing: CUDD_INCLUDE_DIRS)`

Reply:
thanks! fixed with 
4cc0d43